JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1253,4 +1275,45 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(descriptivesQQPlot)
 }
 
+.descriptivesPieChart <- function(dataset, options, variable) {
+  if (options$splitby != "" ) {
+    # return a collection
+    split <- names(dataset)
+
+    plotResult <- createJaspContainer(title=variable)
+    plotResult$dependOn(options="splitby", optionContainsValue=list(variables=variable))
+
+    for (l in split) {
+      plotResult[[l]] <- .descriptivesPieChart_SubFunc(column=dataset[[l]][[.v(variable)]], variable=variable, width=options$plotWidth, height=options$plotHeight, displayDensity = options$distPlotDensity, title = l)
+      plotResult[[l]]$dependOn(optionsFromObject=plotResult)
+    }
+
+    return(plotResult)
+  } else {
+    column <- dataset[[.v(variable)]]
+    aPlot <- .descriptivesPieChart_SubFunc(column=column[!is.na(column)], variable=variable, width=options$plotWidth, height=options$plotHeight, displayDensity = options$distPlotDensity, title = variable)
+    aPlot$dependOn(options="splitby", optionContainsValue=list(variables=variable))
+
+    return(aPlot)
+  }
+}
+
+.descriptivesPieChart_SubFunc <- function(column, variable, width, height, displayDensity, title) {
```

Comment:
```suggestion
.descriptivesPieChart_SubFunc <- function(column, variable, width, height, title) {
```
argument `displayDensity` is unused.

Reply:
thanks, forgot to delete it